coalib/bears/Bear.py

```diff
@@ -32,17 +50,29 @@ def _is_debugged(bear):
     :return:     True if ``debug_bears`` is ``True`` or if bear name specified
                  in ``debug_bears`` setting match with the bear parameter.
     """
-    if not isinstance(bear, Bear):
-        raise ValueError('Positional argument bear is not an instance of '
-                         'Bear class.')
-    if 'debug_bears' not in bear.section:
-        return False
+    setting = _setting_is_enabled(bear, key='debug_bears')
     try:
-        return bool(bear.section['debug_bears'])
+        value = bool(setting)
     except ValueError:
-        pass
-    return bear.name.lower() in (
-            map(str.lower, bear.section['debug_bears']))
+        value = bear.name.lower() in (
+                    map(str.lower, setting))
+    return value
+
+
+def _is_profiled(bear):
+    """
+    Check whether the bear is in profile mode according to its section-settings.
+
+    :param bear: Bear object.
+    :return:     True if ``profile`` is ``True`` or False if ``profile`` is
+                 ``False`` else return directory path specified in ``profile``.
```

Comment:
Maybe always return a directory name, and if it's just `True´, return `os.getcwd()` :+1: Easier to understand.

Reply:
And if the bear is not profiled, return an empty string. (document all that properly)

Reply:
I think False is better instead of `empty string`